src/api_server/swagger/firecracker.yaml

```
@@ -668,6 +746,46 @@ definitions:
         $ref: "#/definitions/TokenBucket"
         description: Token bucket with operations as tokens
 
+  CreateSnapshotParams:
+    type: object
+    required:
+      - snapshot_path
+      - mem_file_path
+    properties:
+      snapshot_path:
+        type: string
+        description: Path to the file that will contain the microVM state.
+      mem_file_path:
+        type: string
+        description: Path to the file that will contain the guest memory.
+      snapshot_type: string
+        enum:
+          - Full
+          - Diff
+        description:
+          Type of snapshot to create. By default, a full snapshot is created.
+      version:
+        type: string
+        description: The snapshot format version.
+
+    LoadSnapshotParams:
+      type: object
+      required:
+        - snapshot_path
+        - mem_file_path
+      properties:
+        snapshot_path:
+          type: string
+          description: Path to the file that contains the microVM state to be loaded.
+        mem_file_path:
+          type: string
+          description: Path to the file that contains the guest memory to be loaded.
+        support_inc_snapshots:
```

Just to be consistent with the above enum I suggest:
```suggestion
        enable_diff_snapshots:
```